packages/coin-tester/src/components/coins/vet/index.tsx

```diff
@@ -0,0 +1,401 @@
+    handleResponse: (response: string) => void
+  ) => {
+    props.setIsLocked(true);
+    try {
+      const response = await request();
+      handleResponse(response);
+    } catch (error: any) {
+      handleResponse(error.message);
+      console.error(error);
+    } finally {
+      props.setIsLocked(false);
+    }
+  };
+
+  const getAddress = async () => {
+    handleState(async () => {
+      const appId = localStorage.getItem('appId');
+      if (!appId) throw new Error('No Appid stored, please register!');
+      return await temp.getAddress(transport!, appPrivateKey, appId, 0);
+    }, setAddress);
+  };
+
+  const signTransaction = () => {
+    useRequest(async () => {
+      const transaction = {
+        chainTag: '0x27',
+        blockRef: connex.thor.status.head.id.slice(4, 18),
+        expiration: web3.utils.toHex(32),
+        clauses: [
+          {
+            to: to,
+            value: web3.utils.toHex(web3.utils.toWei(value, 'ether')),
+            data: '0x',
+          },
+        ],
+        gasPriceCoef: web3.utils.toHex(128),
+        gas: web3.utils.toHex(10000000),
+        dependsOn: '0x',
+        nonce: '0xf2ed7cd2567cabb5',
+      };
+
+      const appId = localStorage.getItem('appId');
+      if (!appId) throw new Error('No Appid stored, please register!');
+      const signTxData = {
+        transport,
+        appPrivateKey,
+        transaction,
+        appId,
+        addressIndex: 0,
+      }
+      console.log("signTxData: ", { signTxData });
+
+      const signedTx = await temp.signTransaction(signTxData);
+
+      const rawTx = getRawTx(transaction)
+      rawTx.push(Buffer.from(handleHex(signedTx), 'hex'))
+      const rawData = rlp.encode(rawTx)
+      console.log("rawData to sync-testnet", '0x' + rawData.toString('hex'))
+
+      // Submit the raw transaction by hand to the test-net.
+      const url = 'https://testnet.veblocks.net/transactions'
+      fetch(url, {
+        method: 'POST',
+        headers: {
+          'Accept': 'application/json',
+          'Content-Type': 'application/json'
+        },
+        body: JSON.stringify({
+          'raw': '0x' + rawData.toString('hex')
+        })
+      }).then(response => {
+        response.text().then(r => { console.log(r) })
+      }).catch(err => {
+        console.log('err', err)
+      })
+
+      return signedTx;
+    }, props).then(setSignedTransaction);
+  }
+
+  const signToken = () => {
+    useRequest(async () => {
+      const transaction = {
+        chainTag: '0x27',
+        blockRef: connex.thor.status.head.id.slice(4, 18),
+        expiration: web3.utils.toHex(32),
+        clauses: [
+          {
+            to: '0x0000000000000000000000000000456E65726779',
+            value: '0x',//this stands for vet, vtho transaction will ignore vet value
+            data: '0xa9059cbb0000000000000000000000007c5Bc2EB55cE9f4Bf9BE2BBEFa1a3c79c8e11AC1' + web3.utils.toHex(web3.utils.toWei(token, 'ether')).slice(2).padStart(64,'0'),
```

Comment:
When signing a token, we'd like to pass a token object instead of a single `data` string.
A token object should include these properties:

```javascript
type TokenInfo = {
  symbol: string; // ex. VTHO
  unit: number; // ex. 18
  contractAddress: string; // 0x0000000000000000000000000000456e65726779
  signature: string; // a signature to validate whether this token address is officially supported by CoolWallet,  just give it "FA".padEnd(144, '0').
}
```

And there is also a need to show a token symbol at CoolWallet Pro, here is the correct display sequence:

VET -> VTHO(token symbol) -> to address -> amount -> Press Button

FYR: https://github.com/CoolBitX-Technology/coolwallet-sdk/blob/a7fc0237e023f0be4073a761088e31d0ea954f14/packages/coin-eth/src/index.ts#L129-L154

Reply:
@hank121314 
All the changes have been committed. Please review. Thanks